### DISCUSSION TITLE
Where does one report false spam reports to Github?

### DISCUSSION THREAD

**Discussion by Chasapple4** [community/community#142491] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Have at least one where there is a false report on a comment being marked as spam, where can I flag it as false?

**Comment by thomassiard** [comment]
If you've encountered a comment that was falsely marked as spam, you can request a review by contacting GitHub Support directly. Here's how:

Submit a Support Ticket: Go to [GitHub Support](https://support.github.com/contact), select "Report Abuse," and explain that the comment was falsely flagged as spam. Include the link to the specific comment or issue for faster review.

GitHub Community Forum: Alternatively, you can post in the GitHub Community Forum under "How to Use Git and GitHub" for additional guidance. However, for specific cases, a support ticket is the best option for direct assistance.

Hope this helps, and good luck!

**Comment by Chasapple4** [comment]
Tried already, as I never got an email about the false report also there is multiple times of false reporting

I even sent logs and screenshots of the issue and that was ignored by the admins.  The issue is over 3 years old.





----
Charlie Schloss

Why Open the Web?

Despite the connecting purpose of the Web, it is not entirely open to all of its users.
When used correctly, HTML documents can be displayed across platforms and devices.
However, many devices are excluded access to Web content.
________________________________
